.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+### Description
+
+Please provide a concise summary of the changes and their motivation.
+
+#### Issues Addressed
+
+List and link all the issues addressed by this PR.
+
+#### Change Type
```

Comment:
Not sure if we need that here if we can use Labels instead/in addition?

Reply:
Well, I would use labels in addition if we need that structuring. Labels are manually attached as well and this is a hint to place the right one on the PR. Granted, you can automate PR labeling, e.g., with [labeler](https://github.com/actions/labeler) or [auto-label-pulls](https://github.com/shiftEscape/auto-label-pulls), but those labeling actions need context information to infer what label(s) to attach. This requires some strict rules or namings applied which we don't have in the `specification` repository.

TL;DR: I'm opting for keeping it so the PR author hints to how this PR is to be processed.